playgrounds/petclinic-maven-sb-2/src/main/scss/petclinic.scss

```
@@ -0,0 +1,214 @@
+}
+
+table td .progress {
+  margin-bottom: 0;
+}
+
+table td.action-column {
+  width: 1px;
+}
+
+.help-block {
+  color: lighten($text-color, 50%); // lighten the text some for contrast
```

Stylelint (17.14.0)

[error] Unknown word // (CssSyntaxError)

(CssSyntaxError)